### PR TITLE
Few fixes and amendments for the boot_options.rst file (#113)

### PR DESCRIPTION
**Pull request by vpodzime**
Some of the options previously documented in the wiki page were missing in our
new official documentation.

### CODE REVIEW

**Comment by bcl** [comment]
oh, bleh. My comments on this ended up on #113

**Comment by vpodzime** [comment]
New version with one typo fixed. 

> oh, bleh. My comments on this ended up on #113

@bcl those comments should be addressed in the most recent version, may I ask for one more review?

File: docs/boot-options.rst [hunk 1]
```diff
@@ -181,7 +201,7 @@ installer-specific options.
 .. ip:
 
 ip
-^^
+^^^
```
Comment (bcl) [inline docs/boot-options.rst]: Extra ^ at the end

File: docs/boot-options.rst [hunk 2]
```diff
@@ -468,6 +503,101 @@ inst.zram
...
+^^^^^^^^^^^^^^
+
+Boot the drives in their existing order, to override the default of booting into
+the newly installed drive on Power Systems servers and EFI systems. This is
+useful for systems that, for example, should network boot first before falling
+back to a local boot.
+
+
+Storage options
+-------------
```
Comment (bcl) [inline docs/boot-options.rst]: Needs a couple more -'s

File: docs/boot-options.rst [hunk 3]
```diff
@@ -468,6 +503,101 @@ inst.zram
...
+as a multipath device. There is no other reason to use this option.
+
+.. warning:: Not for use with actual multipath hardware!  Using this to attempt
+             to install to a single path of a multipath is ill-advised, and not
+             supported.
+
+.. inst.gpt:
+
+inst.gpt
+^^^^^^^^^^
```
Comment (bcl) [inline docs/boot-options.rst]: 2 extra ^ at the end

**Comment by bcl** [comment]
Looks good other than those few comments.

File: docs/boot-options.rst (hunk 1 above)
Comment (vpodzime) [inline docs/boot-options.rst]: Fixing locally, thanks!

**Comment by vpodzime** [comment]
Comments addressed locally. I'm pushing this to master and f23-branch as it is useful for both branches and it predates the F23 branching.

**Pull request closed without merging**